.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: python
+
+python:
+ - "3.6"
+
+sudo: required
+
+install:
+ - pip install -r requirements.txt
+ - pip install -e .
+ - pip install pytest
```

Comment:
Could you add these test-related dependencies in a file called dev-requirements.txt? So we can have a concise list of development requirements. If this suggestion is implemented, lines 11, 12, and 13 would be replaced by ``- pip3 install -r dev-requirements.txt ``

Reply:
also for this, we'd need to remove ``pytest``,``httmock``, and ``pylint`` from the standard requirements file. These packages would now need to be inside ``dev-requirements.txt``

Reply:
Done ;)